resources/views/components/input.blade.php

```diff
@@ -1,17 +1,22 @@
 @props(['name', 'label', 'wrapperClass', 'labelClass'])
```

Comment:
A refactor would be nice so we can get rid of these extra class props?

Reply:
we're going to go over input components in the upcoming stories :)